aodndata/argo/handler.py

```diff
@@ -10,6 +10,14 @@ def __init__(self, *args, **kwargs):
 
         relative_path_root = os.path.join(self._config.pipeline_config['global']['wip_dir'], 'Argo/dac')
         self.resolve_params = {'relative_path_root': relative_path_root}
+        self.include_regexes = ['.*\.nc']
+
+    def preprocess(self):
+        """ set NO_ACTION to non NetCDF files"""
+        self.file_collection.set_publish_types_from_regexes(addition_type=PipelineFilePublishType.NO_ACTION,
```

Comment:
This doesn't read very intuitively to me, since it's kind of inverting the intent of that method, which is a bit clunky and specific (I wrote it early on in the project, and now I look at it it's a bit quirky). I'd be more explicit with something like the filter and set methods, which are nicer, e.g.:

```python
        self.file_collection.\
            filter_by_attribute_id_not('file_type', FileType.NETCDF).\
            set_publish_types(PipelineFilePublishType.NO_ACTION)
```

With the file collection, filter it by non-NetCDF files and set them to NO_ACTION.